test/remote/README.md

```diff
@@ -0,0 +1,92 @@
+# Test remote WDIO REST API
+
+## Boot REST API server
+
+Execute the following command to boot the server to remotely control a WDIO application:
+
+```bash
+npm run start-remote-test-server
+```
+
+> If start succeeds, the server will be running on port 3333
+
+## Start a Chrome application
+
+Send a request to **POST localhost:3333/wdio/application/start** endpoint with the following body:
+
+```json
+{
+    "browserType": "Chrome",
+    "options": {
+        "capabilities": {
+            "browserName": "chrome",
+            "goog:chromeOptions": {
+                "args": ["--no-sandbox"]
+            }
+        }
+    }
+}
+```
+
+## Navigate to URL
+Send a request to POST localhost:3333/wdio/applications/1/navigate endpoint with the following body:
+
+```json
+{
+  "url": "https://www.werfen.com"
+}
+```
+
+## Query a text
+
+Send a request to POST localhost:3333/wdio/applications/1/navigate endpoint with the following body:
```

Comment:
The endpoint route is not correct

Reply:
Endpoint route updated to `localhost:3333/wdio/applications/1/element/query/text`